DecipherNow/gm-fabric-dashboard
# feat: color code legend

Resolves #1495
Resolves #1508 

## Proposed Changes

- Color code the legend
- Fix typo

## How to Test

  - `npm start`
  - Navigate to a chart with multiple values like the JVM heap charts
  - Confirm that the legend is color coded and matches the line chart
  - Open the Go dashboard and ensure that it says "Heap" not "Head"
